test/Microsoft.OpenApi.Tests/Services/OpenApiUrlTreeNodeTests.cs

```diff
@@ -466,5 +466,38 @@ public async Task VerifyDiagramFromSampleOpenAPI()
 
             await Verifier.Verify(diagram);
         }
+
+        public static TheoryData<string, string[], string, string> SupportsTrailingSlashesInPathData => new TheoryData<string, string[], string, string>
+        {
+            // Path, children up to second to leaf, last expected leaf node name, expected leaf node path
+            { "/cars/{car-id}/build/", ["cars", "{car-id}"], "build/", @"\cars\{car-id}\build/" },
+            { "/cars/", [], "cars/", @"\cars/" },
```

Comment:
```suggestion
            { "/cars/{car-id}/build/", ["cars", "{car-id}"], @"build\", @"\cars\{car-id}\build\" },
            { "/cars/", [], "cars/", @"\cars\" },
```

wouldn't this make more sense?

Reply:
That's for you to say.

My interpretation was that `\` was used as as a hierarchy separator, and given the last `/` is not a separator per-se, we might want to keep it as-is to indicate that the last segment is `<whatever>/`.

If you want the changes in, though, that's fine, I'll push them.

Reply:
as long as it doesn't derail the logic/node structure, it'd make sense to be consistent.
@darrelmiller I can't remember why backslashes were originally used here instead of forward slashes, would you mind providing some context please?

Reply:
I can't find out why we used backslash as a separator. It doesn't make sense to me. I tried looking at all the relevant PRs. Maybe @irvinesunday  remembers?

Reply:
alright, thanks for your patience with us @mderriey it seems we suffer from collective amnesia.
In the interest of unblocking this PR, and reducing potential side effects, let's perform the least changes possible, and align everything on backslashes.